tutorials/using-bucket-website-with-mkdocs/mkdocs-example.yml

```diff
@@ -0,0 +1,56 @@
+site_name: My documentation website
+theme:
+  name: 'material'
+  palette: # Palette toggle for light mode
```

Comment:
The `site_url: https://mydomain.org/mysite` component is missing. Either remove it from the mkdocs examples or add a step after the URL retrieval to indicate users need to add it.